ui/src/app/views/home-page/component.html

```diff
@@ -25,7 +25,17 @@ <h3 class="table-header-descriptor">Workspaces</h3>
             </div>
           </ng-template>
           <ng-template #owner>
-            <clr-icon shape="success-standard" class="is-solid"></clr-icon>
+            <div *ngIf="workspaceResponse.accessLevel === workspaceAccessLevel.OWNER else writer">
```

Comment:
This is a pretty good candidate for `ngSwitch`.

Reply:
Sounds good! I'll take a look at that!